### ISSUE TITLE
HAR-5 Define appropriate loss functions

### ISSUE THREAD

**Issue by ILIIH** [ILIIH/Human-action-recognition#6]
Define appropriate loss functions and evaluation metrics for the task of action recognition. Common choices include categorical cross-entropy loss and accuracy.
